lib/charms/alertmanager_k8s/v0/alertmanager_dispatch.py

```diff
@@ -233,22 +234,27 @@ class AlertmanagerProvider(RelationManagerBase):
             charm (CharmBase): the Alertmanager charm
     """
 
-    def __init__(self, charm, relation_name: str = "alerting", api_port: int = 9093):
+    def __init__(
+        self,
+        charm,
+        relation_name: str = "alerting",
+        api_port: int = 9093,
+        *,
+        external_url: Callable = None,
```

Comment:
Shouldn't we be using the external_url arg somewhere in the charm code?

Reply:
We should! Good catch!